smartlearner/preprocessors.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from .interfaces.preprocess import Preprocess
+
+
+class NormalizeFeature(Preprocess):
```

Comment:
We should say what type of normalization it is. According to https://en.wikipedia.org/wiki/Normalization_(statistics) it is "Normalizing errors when population parameters are known. Works well for populations that are normally distributed".